whiskey.yml

```diff
@@ -59,12 +54,38 @@ Build:
     Path: .output\*.zip
 - Zip:
     ArchivePath: .output\Carbon.Core-$(WHISKEY_SEMVER2).zip
-    Path: 
+    Path:
     - Carbon.Core
 - Exec:
     OnlyBy: BuildServer
     Path: appveyor
     Argument: [ 'PushArtifact', '.output/Carbon.Core-$(WHISKEY_SEMVER2).zip', '-DeploymentName', 'GitHub' ]
 
+- Pester:
+    Version: 5.*
+    Configuration:
+        TestResult:
+            Enabled: true
+            OutputPath: .\.output\pester5.xml
+            TestSuiteName: Carbon.Core
+        Run:
+            Path: Tests\*.Tests.ps1
+            ExcludePath:
+            - "*\\Carbon.Core.Tests.ps1"
+            - "*\\ConvertTo-CBase64.Tests.ps1"
+            - "*\\Get-CPowerShellPath.Tests.ps1"
+            - "*\\Invoke-CPowerShell.Tests.ps1"
+            - "*\\Test-COperatingSystem.Tests.ps1"
+            - "*\\Test-CPowerShell.Tests.ps1"

```

Comment:
I can't shake the feeling that we should instead be adding a `pester4/5` suffix to the test filenames instead of always managing these include/exclude lists in the build config.

Reply:
I've avoided this because I try to avoid file renames in source. Depending on the tool, it can be hard to track/follow renames, esp. in code reviews/PRs. If we have a version number anywhere in the path, when a new major version comes out, we'll be renaming files to have the new version in their name. Thoughts? 

This should be something that Pester supports natively, I think, someway of telling it "only run these tests under Pester5, these tests under Pester 6", etc. Maybe I'll submit a feature request.

If we were to put the version number in the filename, what format would we use?

* `.5.Tests.ps1`
* `.P5.Tests.ps1`
* `.Pester5.Tests.ps1`
* `.pester5.Tests.ps1`
* `.pester5.tests.ps1`
* `.Tests.5.ps1`
* `.Tests.P5.ps1`
* `.Tests.Pester5.ps1`
* `.Tests.pester5.ps1`
* `.Tests.pester5.ps1`
* `.Pester5.ps1`

Other? I think I lean toward `.Tests.5.ps1` just for simplicity and easier glob matching. Files matching `*.Tests.ps1` are obviously Pester 4, `*.Tests.5.ps1` Pester 5, etc.

Reply:
> I've avoided this because I try to avoid file renames in source. Depending on the tool, it can be hard to track/follow renames, esp. in code reviews/PRs. If we have a version number anywhere in the path, when a new major version comes out, we'll be renaming files to have the new version in their name. Thoughts?

That's fair, I wasn't considering that.

> If we were to put the version number in the filename, what format would we use?

If we did go this path, I agree with `.Tests.5.ps1` for its simplicity and if not then my next preference would be `.Tests.Pester5.ps1`.